src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,31 @@
+package shop;
+
+import org.junit.jupiter.api.DisplayName;
+import org.testng.annotations.Test;
```

Comment:
Remove testng annotations and use junit only

Reply:
done